combat-trainer.lic

```diff
@@ -333,8 +334,14 @@ class LootProcess
       waitrt?
       bput('drop material', 'you discard it')
     when @rituals['construct']
+      echo "Detected an attempt to do ritual on a construct" if $debug_mode_ct
       game_state.construct(mob_noun)
+    when 'roundtime'
```

Comment:
yea i wanted to know if roundtime was specifically matched, but outputting the result of bput captures that. will remove!